README.md

```diff
@@ -117,9 +138,56 @@ Test results: "test_string_utils.cdc"
 - PASS: testSubstringUntil
 - PASS: testSplit
 - PASS: testJoin
-Coverage: 96.3% of statements
+Coverage: 55.5% of statements
+```
+
+The generated `coverage.json` file is somewhat more elaborate, for integration tests. By viewing its content, we find the following keys:
+
+- `A.01cf0e2f2f715450.ArrayUtils`
+- `A.01cf0e2f2f715450.StringUtils`
+- `A.0ae53cb6e3f42a79.FlowToken`
+- `A.e5a8b7f23e8b548f.FlowFees`
+- `A.ee82856bf20e2aa6.FungibleToken`
+- `A.f8d6e0586b0a20c7.FlowServiceAccount`
+
+and some other locations. The ones starting with `s.*` are scripts, while the ones starting with `t.*` are transactions.
+
+The `ArrayUtils` smart contract is imported by `StringUtils`, that's why it was deployed on the integration tests, and that's why it is included in the resulting coverage report.
+
+For viewing the coverage report of the `StringUtils` smart contract, we can just consult the value of the `A.01cf0e2f2f715450.StringUtils` key, in the `coverage.json` file.
+
+The rest of the keys are system contracts that are always available in the Flow Emulator, which is utilized as the backend implementation for integration tests.
+
+## For Emulator
+
+It is also possible to view code coverage through the emulator, outside the context of testing.
+
+All we have to do is start the emulator with the necessary flag (`coverage-reporting`):
+
+```bash
+flow emulator --storage-limit=false --coverage-reporting -v
 ```
 
-Look at the files `ArrayUtils.cdc` (smart contract) and `test_array_utils.cdc` (tests for the smart contract). For the `ArrayUtils.range` method, we have omitted in purpose the branch where `start > end`. It is left as an exercise to the reader.
+With this, we can use our browser and visit http://localhost:8080/emulator/codeCoverage.
+
+This code coverage report will reflect every interaction with the emulator. For example, we can deploy contracts to the emulator, run scripts/transactions against them and view the results:
+
+```bash
+flow deploy contracts --network=emulator
+
+flow scripts execute foo_contract_scripts.cdc --network=emulator
+```
+
+![Emulator Code Coverage](./emulator-code-coverage.png)
+
+We can also flush/reset the collected code coverage report, with:
+
+```bash
+curl -XPUT 'http://localhost:8080/emulator/codeCoverage/reset'
+```
+
+Which results in the following:
+
+![Code Coverage Reset](./code-coverage-reset.png)
 
-Also, look at the files `StringUtils.cdc` (smart contract) and `test_string_utils.cdc` (tests for the smart contract).
+All of the keys have disappeared, except for the `A.f8d6e0586b0a20c7.FlowServiceAccount`, which is a system contract that runs frequently some operations.
```

Comment:
```suggestion
All of the keys have disappeared, except for `A.f8d6e0586b0a20c7.FlowServiceAccount`, which is a system contract that is essential to the operations of Flow.
```

Reply:
Updated in https://github.com/Build-Squad/flow-code-coverage/pull/1/commits/385067c991608e7a37eba742bd857fab136ebeb0